packages/nouns-contracts/contracts/governance/NounsDAOAdmin.sol

```diff
@@ -537,12 +546,10 @@ library NounsDAOAdmin {
     }
 
     /**
-     * @notice Admin function for zeroing out the state variable `voteSnapshotBlockSwitchProposalId`
-     * @dev We want to zero-out this state slot so we can remove this temporary variable from contract code and 
-     * be ready to reuse this slot.
+     * @notice Admin function for setting the canonical Nouns Fungible Token address
      */
-    function _zeroOutVoteSnapshotBlockSwitchProposalId() external onlyAdmin {
-        ds().voteSnapshotBlockSwitchProposalId = 0;
+    function _setNounsFungibleToken(address newNounsFungibleToken) public onlyAdmin {
+        ds().nounsFungibleToken = newNounsFungibleToken;
```

Comment:
add event?

Reply:
I've been thinking we don't really use these events ever; esp this param will be set once and probably never again, unlike other gov params. can you please share your thoughts on why add an event here?

Reply:
the general rule of thumb I use is that whenever storage is modified, an event should be fired.
I think there should be a stronger reason for not-adding than for adding

Reply:
added an event